install/kubernetes/cilium/templates/cilium-agent/validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1alpha1/ValidatingAdmissionPolicy" }}
+---
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "http-tcp"
```

Comment:
Let's use name here that indicates this comes from Cilium, e.g. `cilium-validate-network-policies` (I'm assuming we want to use additional validations later too)